### DISCUSSION TITLE
Web site with port number can not get the site icon

### DISCUSSION THREAD

**Discussion by nbihecfc** [dani-garcia/vaultwarden#4022] [Ideas]
When the URL contains :, such as 'https://github.com:9999', the site icon is not available and the default icon is displayed.
Is it possible to fix this bug in the next minor version?
Without the icon, the display is not as effective as it should be.

**Comment by BlackDex** [comment]
I'm afraid that we can't fix this in Vaultwarden.
The clients need to send this, but they do not and only send the host/domain.
